oml/const.py

```diff
@@ -101,6 +102,17 @@ def get_cache_folder() -> Path:
 
 IMAGE_EXTENSIONS = ["jpg", "jpeg", "JPG", "JPEG", "png"]
 
-# hydra provides ability to set its behaviour for convinient backwards compatibility
+# hydra provides ability to set its behaviour for convenient backwards compatibility
 # no matter what the current version is
 HYDRA_BEHAVIOUR = "1.1"
+
+# Audio specific constants
+MOCK_AUDIO_DATASET_PATH = CACHE_PATH / "mock_audio_dataset"
+MOCK_AUDIO_DATASET_URL_GDRIVE = "https://drive.google.com/drive/folders/1NcKnyXqDyyYARrDETmhJcTTXegO3W0Ju"
+MOCK_AUDIO_DATASET_MD5 = "d983fb30e6cbcf0f79aaf88d2ea0ec83"
```

Comment:
I got a different hash: `45629afad9ff28f9148f3ea60f659d18`

Reply:
For me `45629afad9ff28f9148f3ea60f659d18` too with updated structure

Reply:
@DaloroAT Aman said it was `.DS_Store` file locally)

Reply:
Haha 😁 💩